doc/api/debugger.md

```diff
@@ -1,5 +1,7 @@
 # Debugger
 
+<!--introduced_in=v0.10.0-->
```

Comment:
I don't think this is correct either? https://nodejs.org/dist/v0.9.12/docs/api/debugger.html